### DISCUSSION TITLE
VivaldiFox no compatible

### DISCUSSION THREAD

**Discussion by Neikon** [Godiesc/firefox-one#7] [Ideas]
Sería posible que este tema fuera compatible con VivaldiFox para obtener los colores de la interfaz desde la web o hacer algo así desde el propio tema (aunque no sé si eso es posible)

**Comment by Godiesc** [comment]
hi,
pues sería posible modificando el tema en sí, por ejemplo el tema tiene establecido el color para la pestaña seleccionada y la barra de herramientas y debe interferir con la extension de vivaldi-fox, podrías empezar eliminando la linea 20 y 40 de "one_root.css" para temas oscuros y la linea 129 and 149 para temas claros, ten en cuenta que esos colores los uso para dar estilo en otras partes del navegador y puede dar un resultado extraño.
